Services/Administration/OJS.Services.Administration.Business/IParticipantsBusinessService.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace OJS.Services.Administration.Business;
+
+using SoftUni.Services.Infrastructure;
+using System.Threading.Tasks;
+
+public interface IParticipantsBusinessService : IService
+{
+    Task UpdateTotalScoreSnapshotOfParticipants();
+
+    Task RemoveParticipantMultipleScores();
```

Comment:
Document these methods with ///summary as it is not clear why and what they do.
Don't forget to mention that we leave only the best score untouched.

Reply:
It's fixed.